src/login/login.jsx

```diff
@@ -25,52 +25,50 @@ const Login = () => {
   };
 
   return (
-    <div>
-      {userLoggedIn && <Navigate to={"/home"} replace={true} />}
-      <main>
+    <div className="loginContainer">
+      <div className="center">
+        {userLoggedIn && <Navigate to={"/home"} replace={true} />}
         <div>
-          <div>
-            <h3>Welcome Back</h3>
+          <h1>Welcome Back</h1>
+        </div>
+        <form onSubmit={onSubmit}>
+          <div className="txtField">
+            <input
+              type="text"
+              autoComplete="email"
+              required
+              value={email}
+              onChange={(e) => {
+                setEmail(e.target.value);
+              }}
+            />
+            <span></span>
```

Comment:
What is this span for? How does it the component look like if removed?

Reply:
I think I was using it for something and I forgot to delete it

Reply:
I incorporated the span into the css check login.css for more details